component/container-runtime.jsonnet

```diff
@@ -4,17 +4,21 @@ local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 
+local machineConfigPools = import 'machine-config-pools.libsonnet';
+
 local params = inv.parameters.openshift4_nodes;
 
+local mergedConfigs = machineConfigPools.ContainerRuntimeConfigs + com.makeMergeable(params.containerRuntimeConfigs);
+
 local containerRuntimeConfigs = [
   kube._Object('machineconfiguration.openshift.io/v1', 'ContainerRuntimeConfig', nodeGroup) {
     metadata+: {
       labels+: common.DefaultLabels,
     },
```

Comment:
Nit: For the other resources the PR introduces wrapper functions for the `kube._Object()`, but here it doesn't, would be nice to have this consistent.